src/ray/core_worker/core_worker.cc

```diff
@@ -239,6 +246,37 @@ void CoreWorkerProcess::RunTaskExecutionLoop() {
   instance_.reset();
 }
 
+void CoreWorkerProcess::RunStatsService() {
+  absl::MutexLock lock(&CoreWorkerProcess::stats_initialization_mutex_);
+  enable_stats_count_++;
```

Comment:
Why not put the lock and count thing inside `ray::stats::Init`?

Reply:
It's just used for core worker and multiple core worker could be initialized in one process, but raylet/gcs can not be.

Reply:
Actually, stats init also return diretctly if it's initialized and marked in flag.